### DISCUSSION TITLE
Welcome to Your-First-Open-Source-Project Discussions!

### DISCUSSION THREAD

**Discussion by melvincwng** [Your-First-Open-Source-Project/Your-First-Open-Source-Project#3] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nosacodes4fun** [comment]
So what is this Open source project about?

**Reply by melvincwng** [reply]
@nosacodes4fun , this is a beginner friendly repository to introduce newcomers on how to make open source contributions.

Just simply add your GitHub name and user profile link in CONTRIBUTORS.md to make your first open source contribution today.

:)

**Reply by nosacodes4fun** [reply]
Done this. Waiting to merge

**Comment by SIGEIV** [comment]
A crypto mixer

**Comment by axewhyzed** [comment]
Thanks for making this repo for all the beginners out there

**Reply by melvincwng** [reply]
Your welcome @axewhyzed, glad to help 😊

**Comment by MegaCloudsLLC** [comment]
Well, I need to know how to utilize the open source opportunities. I wish to contribute

**Reply by melvincwng** [reply]
https://github.com/Your-First-Open-Source-Project/start-here

**Comment by MegaCloudsLLC** [comment]
Where do we kick off from?

**Reply by melvincwng** [reply]
https://github.com/Your-First-Open-Source-Project/start-here

**Comment by SUDEEP-M-SHETTY** [comment]
Any beginner friendly AWS Educational Resources for people new to it?

**Reply by melvincwng** [reply, marked as answer]
Take a look at these available AWS resources for our community: https://github.com/yfosp/aws-resources
